### DISCUSSION TITLE
feat:  Plans to package pre-built binaries with releases ?

### DISCUSSION THREAD

**Discussion by Imaclean74** [eza-community/eza#186] [Q&A, Howto, Help] [answered]
similar to the [exa-linux-x86_64-musl-v0.10.1.zip](https://github.com/ogham/exa/releases/download/v0.10.1/exa-linux-x86_64-musl-v0.10.1.zip)  packages packaged as part of exa releases. These are very useful for those of us packaging things in a corporate environment without access to external package repos.

**Comment by cafkafk** [comment, marked as answer]
Yes, I'm actively working on packaging binary releases, and if all goes well, we'll see binaries for supported platforms in one of the coming releases.

Related: #101

**Reply by ariasuni** [reply]
I wasn’t sure where to put this, but here’s a link to various projects and their CIs to build ARM64 and Apple Silicon executables that we can use as an inspiration for eza:
https://github.com/ogham/exa/issues/414#issuecomment-1110630237

**Reply by cafkafk** [reply]
thanks, I'll check that out!

**Reply by Imaclean74** [reply]
I see we have both Linux and windows binaries as of the last release. Nice one ! I've already uploaded those for use on our internal systems.
